cmd/gangway/handlers.go

```diff
@@ -313,13 +336,19 @@ func commandlineHandler(w http.ResponseWriter, r *http.Request) {
 		ClusterCA:    string(caBytes),
 		HTTPPath:     cfg.HTTPPath,
 	}
-
-	generateKubeConfig("kubeconfig.tmpl", info)
-	serveTemplate("commandline.tmpl", info, w)
+	return info
 }
 
 func kubeConfigHandler(w http.ResponseWriter, r *http.Request) {
+	info := generateInfo(w, r)
```

Comment:
We should handle the case where info == nil. Otherwise, we'll write to the ResponseWriter twice.